Wangscape/NormLPQ.cpp

```diff
@@ -0,0 +1,34 @@
+#include "NormLPQ.h"
+#include <cmath>
+
+
+
+NormLPQ::NormLPQ(double p, double q):
+    Module(GetSourceModuleCount())
+{
+    setPQ(p, q);
+}
+
+double NormLPQ::GetValue(double x, double y, double z) const
+{
+    double ax = std::abs(x);
+    double ay = std::abs(y);
+    double az = std::abs(z);
```

Comment:
I don't think so. Libnoise is explicitly built to work with 3D noise, and even if we altered the API of the Wangscape `Module`s to accept a struct, the libnoise `Module`s would still be used through `GetValue(x,y,z)`.

We could start using `noise::model::Plane` to remove the `z` from some of the calls. The `Module`s would still need to accept `z` as an argument.